package.json

```diff
@@ -24,8 +24,8 @@
   "homepage": "https://github.com/digitalbazaar/bedrock-did-io",
   "dependencies": {
     "@digitalbazaar/did-io": "^1.0.0",
-    "@digitalbazaar/did-method-key": "^1.0.0",
-    "did-veres-one": "14.0.0-beta.0",
+    "@digitalbazaar/did-method-key": "^1.2.0",
+    "did-veres-one": "veres-one/did-veres-one#support-2018",
```

Comment:
It was unclear to me from the `did-veres-one` changelog if the additional breaking changes were from `beta.0` to `beta.1`, but if there are, it would be best to release this as major/breaking.

Reply:
Done, thanks.